### ISSUE TITLE
Failed autowithdrawals show up in wallet logs as 'OK'

### ISSUE THREAD

**Issue by huumn** [stackernews/stacker.news#1069]
Testing #1042, regardless of whether an autowithdraw succeeds or fails I get an `OK` message in the wallet logs

<img width="743" alt="Screenshot 2024-04-14 at 5 01 28 PM" src="https://github.com/stackernews/stacker.news/assets/34140557/a2429f5e-ed64-4bbe-bfff-7938222ef0a4">

Can you tell which of these autowithdraws failed?

(All of the autowithdrawals failed except the last one.)

**Comment by ekzyis** [comment]
Mhh, are you sure that these failed?

Failed autowithdrawals shouldn't show up in the logs at all yet since only the creation of the withdrawal is logged but not the actual withdrawal via lightning. I forgot to mention that in #994.

I also just tested again and my withdrawal failed because no route was found and it doesn't show up in the logs. I crosschecked with wallet history where it does show up.

So I think the logs are just confusing since errors don't show up. So logging the errors is what this ticket is about afaict.
